packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -343,9 +360,13 @@ class TransferEngine<
       await this.transferLinks();
       await this.transferConfiguration();
 
+      this.#emitTransferUpdate('finish');
```

Comment:
If we consider the transfer started before the bootstrap & the init, shouldn't we wait for the close lifecycle to be run before sending the finish event?

Reply:
Yeah, I was considering it a success if the actual data transfered, but I can see it either way. If we're still able to rollback in close, it should definitely be after, so I've moved it.